src/app/community-list-page/community-list/community-list.component.ts

```diff
@@ -111,19 +111,18 @@ export class CommunityListComponent implements OnInit, OnDestroy {
   getNextPage(node: FlatNode): void {
     this.loadingNode = node;
     if (node.parent != null) {
-      if (node.id === 'collection') {
+      if (node.id.startsWith('collection')) {
```

Comment:
What is the reason behind the exact matches vs startWith matches changes here?

Reply:
I see this comment in the PR:
>The id of the show more node added to the tree must be unique otherwise it is placed at the wrong level and clicking pages the wrong page (level). To make the node id unique, it was appended with hyphen and a newly generated UUID. The condition during getNextPage was updated to match node.id startsWith collection or community.

Is it that the nested structures are appending to the original id such that startsWith is needed?

Reply:
The node ids added to the cdk tree should be unique. Otherwise, placement in the tree and click on duplicate has undesired effect. It would show more of the first show more node list containment.

Reply:
In our nested community/collection structure, the tree ended up with many show more nodes with id either `community` or `collection` pending of the list it was paginating as they are typed per datasource service.